spec/javascripts/FrameSpec.js

```diff
@@ -0,0 +1,16 @@
+describe('Frame', function(){
+  it("is over after 2 rolls", function(){
+    var frame = new Frame();
+    expect(frame.isOver()).toEqual(false);
+    frame.roll(0);
+    frame.roll(0);
+    expect(frame.isOver()).toEqual(true);
+  });
+
+  it("strike ends the frame", function(){
+    var frame = new Frame();
+    expect(frame.isOver()).toEqual(false);
+    frame.roll(10);
+    expect(frame.isOver()).toEqual(true);
```

Comment:
Nice tests, really clear! This is a bit picky but maybe to double check that your functions are doing what they are doing, could you test the same thing another way? ( Literally, this is me just reaching for things to comment about this fab code, idk what I'm doing hahahaha)